### PR TITLE
code changes to tie ingestion and validation

### PR DESCRIPTION
**Pull request by yogesh266**
## Description
code changes to tie ingestion and validation
Screenshot 2023-07-07 at 10 24 59 AM
Screenshot 2023-07-07 at 10 27 03 AM

### CODE REVIEW

File: ads/feature_store/common/utils/utility.py [hunk 1]
```diff
@@ -116,6 +117,60 @@ def validate_delta_format_parameters(
...
+    from tabulate import tabulate
+
+    table_headers = (
+        ["expectation_type"] + list(statistics.keys()) + ["ingestion_status"]
+    )
+
+    table_values = [expectation_type] + list(statistics.values()) + [ingestion_status]
+
+    logger.info(
+        "Validation Summary \n"
```
Comment (KshitizLohia) [inline ads/feature_store/common/utils/utility.py:163]: is it possible to also display summary of all the validation rules as part of the table?

**Comment by KshitizLohia** [comment]
let's update the pr with unit tests and integration tests results

File: ads/feature_store/common/utils/utility.py (hunk 1 above)
Comment (yogesh266) [reply ads/feature_store/common/utils/utility.py:168]: Done

**Review by KshitizLohia — APPROVED** [review]
LGTM

**Pull request merged**